joomla-projects/webservices
# New MVC for webservices component

This is an initial approach to new MVC move. All controllers seems to be working fine but test on ajax is needed. I changed the old way to call ajax to a new way using an specific controller (AjaxController.php). Save option seems to be working fine, creating the xml file and wsdl too, but also needs a detailed test.

All comments are very welcome.
## Test instructions

1) Get GIT files
2) Install Joomla!
3) link or copy $GIT/component/admin to $WWWROOT/administrator/components/com_webservices 
4) copy $GIT/component/webservices.xml to $WWWROOT/administrator/components/com_webservices  (should we copy it to $GIT/component/admin?)
5) Go to discover on Joomla! installation and install the extension
6) Edit $WWWROOT/administrator/components/com_webservices/webservices.php line 17 and put the real path of the GIT files
7) Edit $GIT/config.json and set your database credentials
